ksql-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedSerdeFactory.java

```diff
@@ -35,6 +38,15 @@
 @Immutable
 public class KsqlDelimitedSerdeFactory implements KsqlSerdeFactory {
 
+  private static final Delimiter DEFAULT_DELIMITER = Delimiter.parse(',');
+
+  private final CSVFormat csvFormat;
+
+  public KsqlDelimitedSerdeFactory(final Optional<Delimiter> delimiter) {
+    this.csvFormat =
+        CSVFormat.DEFAULT.withDelimiter(delimiter.orElse(DEFAULT_DELIMITER).getDelimiter());
```

Comment:
nit: strange formatting...  blank line etc.

I'd probably go with:

```java
    this.csvFormat = CSVFormat.DEFAULT
        .withDelimiter(delimiter.orElse(DEFAULT_DELIMITER).getDelimiter());
```

It it were me.